src/Calendar.js

```diff
@@ -433,8 +435,8 @@ const __class = declare('argos.Calendar', [ _Widget, _ActionMixin, _Templated],
 
     this.titleText = this.options.label ? this.options.label : this.titleText;
     this.showTimePicker = this.options && this.options.showTimePicker;
-    this.date.selectedDateMoment = moment((this.options && this.options.date) || moment());
-    this.date.todayMoment = moment();
+    this.date.selectedDateMoment = moment((this.options && this.options.date) || moment().clone());
+    this.date.todayMoment = moment().clone();
```

Comment:
You shouldn't have to clone new instances of moment created by `moment()`. You should only clone when modifying an existing instance.